pkgs/tools/misc/colorless/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+, coreutils
+, dash
+, gnused
+, less
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colorless";
+  version = "109";
+
+  src = fetchurl {
+    url = "http://software.kimmo.suominen.com/${pname}-${version}.tar.gz";
+    sha256 = "039a140fa11cf153cc4d03e4f753b7ff142cab88ff116b7600ccf9edee81927c";
+  };
+
+  makeFlags = [
+    "TOOLPATH=${coreutils}/bin:${gnused}/bin:${less}/bin"
+    "PREFIX=${placeholder "out"}"
```

Comment:
```suggestion
    "PREFIX=$(out)"
```

Reply:
Thank you! In browsing other code the distinction between curly braces and parenthesis was lost on me (i.e. `${out}` was resulting in an error, and then I saw `${placeholder "out"}` in some other code).

Reply:
They both work, but placeholder is kind of a last resort if the plain old environment variables does not work.